### PR TITLE
docs: improve auto routing

### PR DESCRIPTION
**Pull request by kenjis**
**Description**
- improve description for Default Controller in Auto-Routing
- update for consistency

**Checklist:**
- [x] Securely signed commits
- [] Component(s) with PHPDoc blocks, only if necessary or adds value
- [] Unit testing, with >80% coverage
- [x] User guide updated
- [] Conforms to style guide

### CODE REVIEW

File: user_guide_src/source/incoming/controllers.rst [hunk 1]
```diff
@@ -274,16 +274,21 @@ Your method will be passed URI segments 3 and 4 (``'sandals'`` and ``'123'``):
...
 
-.. note:: If there are more parameters in the URI than the method parameters,
+.. important:: If there are more parameters in the URI than the method parameters,
     Auto Routing (Improved) does not execute the method, and it results in 404
     Not Found.
 
+Default Controller
+==================
+
+The Default Controller is a special controller that is used when a URI end with
```
Comment (michalsn) [inline user_guide_src/source/incoming/controllers.rst:284]:
```suggestion
The Default Controller is a special controller that is used when a URI ends with
```

**Review by michalsn — COMMENTED** [review]
This question is not very important, but why are we changing `folder` to `directory` everywhere? Do we stick to this convention on other pages? I have nothing against it, though.

**Comment by kenjis** [comment]
This is an explanation of subdirectories in Controllers, and there is no need to use the two terms *folder* and *directory*. Suddenly changing what we call a directory to a folder may confuse beginners.

I do not intend to change all *folders* in the user guide to *directories*.

**Review by michalsn — APPROVED** [review]

**Comment by kenjis** [comment]
Thank you! @michalsn

**Pull request merged**